### PR TITLE
Remove last usages of Listing/ListingRow React component

### PR DESCRIPTION
**Pull request by KKoukiou**
Needs  some text fixes

### CODE REVIEW

**Review by marusak — CHANGES_REQUESTED** [review]
Nice work!
I noticed only one issue:
Current master:
currentmaster
This PR:
nowok

**Comment by KKoukiou** [comment]
@marusak fixed.
Screen Shot 2020-10-13 at 10 45 32

File: pkg/storaged/side-panel.jsx [hunk 1]
```diff
@@ -104,7 +107,7 @@ export class SidePanelRow extends React.Component {
                     {this.props.actions}
                 </div>);
         else if (client.path_jobs[job_path])
-            decoration = <div className="sidepanel-row-decoration spinner spinner-sm" />;
+            decoration = <Spinner size="sm" />;
         else if (client.path_warnings[job_path])
             decoration = <div className="sidepanel-row-decoration pficon pficon-warning-triangle-o" />;
```
Comment (marusak) [inline pkg/storaged/side-panel.jsx:112]: It is this line that I get there rendered.
Still looks the same though :/

Comment (KKoukiou) [reply pkg/storaged/side-panel.jsx:112]: Are you sure :) ?

I recompiled it and it looks like this:

Screen Shot 2020-10-13 at 16 28 15

Comment (marusak) [reply pkg/storaged/side-panel.jsx]: Hmm, maybe I failed to rebuild it? I would try again, but there are not the correct changes

**Comment by KKoukiou** [comment]
Fixed merge conflict and repushed.

**Comment by KKoukiou** [comment]
@marusak Can you recheck this in case you did not build dist properly for the comment above?

**Comment by marusak** [comment]
@KKoukiou still the same behaviour. Are you sure it works on your side, maybe you did not commit something?

**Comment by KKoukiou** [comment]
@marusak thanks for helping debug this, 
I think it's ok now :)
Screen Shot 2020-10-14 at 13 42 17

**Comment by KKoukiou** [comment]
> @marusak thanks for helping debug this,
> I think it's ok now :)
> Screen Shot 2020-10-14 at 13 42 17

tests fail of course, this one does not want to get merged I think.

**Comment by marusak** [comment]
It has conflict, can you please rebase, then I'll check again but I think we are there :)

**Review by marusak — APPROVED** [review]
Thanks!

**Pull request merged**